docs/models/clip.md

```diff
@@ -0,0 +1,128 @@
+# CLIP
+
+
+The [CLIP](https://arxiv.org/abs/2103.00020) features are extracted at each frame of the provided video.
+CLIP (Contrastive Language-Image Pre-Training) is a neural network trained on a variety of (image, text) pairs.
+We use CLIP's official augmentations and extract vision features from its image encoder.
+The implementation uses the [openAI CLIP](https://github.com/openai/CLIP).
+The extracted features are going to be of size `num_frames x 512`.
+We additionally output timesteps in ms for each feature and fps of the video.
+
+---
+
+## Set up the Environment for CLIP
+Setup `conda` environment. Requirements are in file `conda_env_torch_zoo.yml`
+```bash
+# it will create a new conda environment called 'torch_zoo' on your machine
+conda env create -f conda_env_torch_zoo.yml
+```
+
+---
+
+## Minimal Working Example
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/13BamfEryOz0z_M-zWw5MHdf4lVvBtwj0?usp=sharing)
+
+Activate the environment
+```bash
+conda activate torch_zoo
+```
+
+and extract features at 1 fps from `./sample/v_GGSY1Qvo990.mp4` video and output results.
+```bash
+python main.py \
+    feature_type=clip \
+    model_name=ViT-B/32 \
```

Comment:
TODO for myself: test this on linux. 
* Will it fail because of the special characters? 
* Should we use ""?

Reply:
There may be bugs due to some special characters, but I guess the model name in general should not have those special characters.

Reply:
It is actually ok. However, I decided to surround these arguments with `"` in the docs.